.changeset/ten-impalas-hope.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-events-backend': minor
+'@backstage/plugin-events-node': minor
```

Comment:
```suggestion
'@backstage/plugin-events-backend': patch
'@backstage/plugin-events-node': patch
```

As those are still non-stable packages, `patch` should be enough here as there is no breaking change.